docs/optimization_guide/dldt_deployment_optimization_denormals.md

```diff
@@ -0,0 +1,27 @@
+# Denormals Optimization {#openvino_docs_deployment_optimization_guide_denormals}
```

Comment:
Do we really need a dedicated page for this property? can it just be one of the options supported by CPU plugin? W/o explicit page
Like of the properties here https://docs.openvino.ai/nightly/openvino_docs_OV_UG_supported_plugins_CPU.html#supported-properties
The description in this document can be moved to the property documentation as well, it's good extensive description.

@dmitry-gorokhov how do you think?

Reply:
I agree. Separate page seems to be overkill there. So lets move it to the plugin page.
I would also propose the following:
1. From my perspective the property itself should be cpu specific. Current implementation via FTZ abd DAZ flags affects only CPU. Other plugins are not affected and seems to be will not provide corresponding target specific implementations in the nearest future. So I would prefer to keep this as cpu specific property and make in common only once other plugins will require it. 
See this example on how GPU [implements and documents](https://docs.openvino.ai/latest/groupov_runtime_ocl_gpu_prop_cpp_api.html#doxid-group-ov-runtime-ocl-gpu-prop-cpp-api-1ga4545149544127b7f82b5d673b8a5a017) specific properties: 
2. Lets have a small section inside main [CPU plugin page](https://docs.openvino.ai/latest/openvino_docs_OV_UG_supported_plugins_CPU.html) that have "optimization guide" and recommends to try denormal key (with code sample) and warns a user about possible accuracy degradation ([example](https://docs.openvino.ai/latest/openvino_docs_OV_UG_supported_plugins_GPU.html#gpu-performance-checklist-summary)).

Reply:
have done! thanks.